RELEASENOTES/v2.0.0.rc-1.MD

```diff
@@ -0,0 +1,64 @@
+  - Updated libraries for Notation
+    - Both Notation-go and Notaiton-core-go libraries have been updated.
+    - Users can directly integerate with notation-go for signing and verification APIs. These APIs are made configurable to support verification of pre-fetched signatures one at a time.  
+*This is the first supported release from the Notary v2 community. Refer release definitions [here](https://github.com/notaryproject/notation/blob/main/RELEASE_MANAGEMENT.md). We suggest using it for testing and providing feedback to the Notary v2 working group only*
+
+## List of feature/capabilities added with this release
+For detailed notes on each of the individual repositories (NotaryProject, Notation (for reference CLI implementation) and Notation libraries) users can refer to the individual release notes. A summary view is provided in this document.
+
+### Notation CLI Client
+Notation is a standalone CLI client based on Notary v2 specs. It supports signing, verifying, and storing signatures in registries supporitng signatures as references to images.
+
+In this release of Notation command, the following CLI commands are supported. 
+  - sign        Sign artifacts
+  - verify      Verify OCI artifacts
+  - certificate Manage certificates in trust store for verifying 
+  - key         Manage keys used for signing
+  - list        List signatures of the signed artifact
+  - login       Login to registry
+  - logout      Log out from the logged in registries
+  - plugin      Manage plugins  ( Only notation plugin list commmand is support in this release)
+  - version     Show the notation version information
+  - help        Help about any command
+
+As part of this release, users have to create a trust policy configuration file before verifying arifacts. 
+
+To start using Notation Refer: https://notaryproject.dev/docs/quickstart/ 
+
+### Notation Library 
+Written in the GO language the Library is divided up into two separate repos - [Notation-go](https://github.com/notaryproject/notation-go) and [Notation-core-go](https://github.com/notaryproject/notation-core-go)
+
+## List of fixed issues
+Refer individual release notes.
+
+## Roadmap Guidance
+*This is not an exhaustive list. Users should refer to the Notary v2 roadmap at https://github.com/notaryproject/roadmap#readme for seeing the plan for subsequent releases and refer to individual subprojects (repositories) releases notes for details*
+
+1. Alternate signature storage schemes which can use OCI Distribution 1.0 based registries ("fallback support") is planned for the next release candidate
+2. Revocation support via Notation client will come in a future release. Spec update for revocation is planned for the next release candidate
+
+
+### Backwards Compatibility
+This version of Notation client is not compatible with any of the prior releases.
```

Comment:
This version works only with registries compatible with OCI IMAGE 1.1 spec

Reply:
fixed